packages/client/src/components/ProductPropType/ProductPropType.component.js

```diff
@@ -0,0 +1,6 @@
+import PropTypes from 'prop-types';
```

Comment:
This is good but I wouldn't put it into `/components` since it's not a component, find some other place to put it please :)

Reply:
I have moved to outside component and Its in src directory.